### PR TITLE
android: implemented one hand ui for chat list screen

### PR DESCRIPTION
**Pull request by diogofcunha**
Adds a usable one hand UI on android.

A recording of the behaviour:

https://github.com/user-attachments/assets/64aed770-ae08-4211-87d9-f0982f7e7866

### CODE REVIEW

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt [hunk 1]
```diff
@@ -91,13 +116,20 @@ fun ChatListView(chatModel: ChatModel, settingsState: SettingsViewState, setPerf
     drawerGesturesEnabled = appPlatform.isAndroid,
     floatingActionButton = {
       if (searchText.value.text.isEmpty() && !chatModel.desktopNoUserNoRemote && chatModel.chatRunning.value == true) {
+        var bottom = DEFAULT_PADDING
+        if (oneHandUI.state.value) {
+          bottom = DEFAULT_BOTTOM_PADDING
+        } else {
+          bottom -= 16.dp
+        }
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt:124]:
```suggestion
        val bottomPadding = if (oneHandUI.state.value) DEFAULT_BOTTOM_PADDING else DEFAULT_PADDING - 16.dp
```

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt [hunk 2]
```diff
@@ -91,13 +116,20 @@ fun ChatListView(chatModel: ChatModel, settingsState: SettingsViewState, setPerf
...
+
         FloatingActionButton(
           onClick = {
             if (!stopped) {
               if (newChatSheetState.value.isVisible()) hideNewChatSheet(true) else showNewChatSheet()
             }
           },
-          Modifier.padding(end = DEFAULT_PADDING - 16.dp + endPadding, bottom = DEFAULT_PADDING - 16.dp).size(AppBarHeight * fontSizeSqrtMultiplier),
+          Modifier.padding(end = DEFAULT_PADDING - 16.dp + endPadding, bottom = bottom).size(AppBarHeight * fontSizeSqrtMultiplier),
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt:132]: something's wrong with floating action button padding in both modes - when it's clicked it's not shown over the initial button shape - the opened menu together with button should be padded so it's on the same height

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt [hunk 3]
```diff
@@ -226,10 +263,26 @@ private fun ChatListTopBar(stopped: Boolean) {
...
   DefaultTopAppBar(
+    navigationButton = {
+      if (chatModel.users.isEmpty() && !chatModel.desktopNoUserNoRemote) {
+        NavigationButtonMenu { scope.launch { if (drawerState.isOpen) drawerState.close() else drawerState.open() } }
+      } else {
+        val users by remember { derivedStateOf { chatModel.users.filter { u -> u.user.activeUser || !u.user.hidden } } }
+        val allRead = users
+          .filter { u -> !u.user.activeUser && !u.user.hidden }
+          .all { u -> u.unreadCount == 0 }
+        UserProfileButton(chatModel.currentUser.value?.profile?.image, allRead) {
```
Comment (spaced4ndy) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt:277]: in regular mode (one hand UI off) user picker is incorrectly located in the bottom

Comment (diogofcunha) [reply apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt:283]: fixed

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt (hunk 1 above)
Comment (diogofcunha) [reply apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt:124]: if you don't mind will skip these as per offline chat with EP. we will both remove the anymation and also stop having this button altogether in one hand

**Pull request merged**